haproxy/values.yaml

```diff
@@ -0,0 +1,31 @@
+replicaCount: 1
+
+image:
+  repository: docker.io/haproxytech/haproxy-debian
+  tag: 2.1
+  pullPolicy: IfNotPresent
+
+ingress:
+  enabled: false
+
+service:
+  port: 30636
+  type: ClusterIP
+
+haproxy:
+  config: ldap-tls
+  ldapTls:
+    certificatePath: /etc/ssl/certs/ca-certificates.crt
+    backend: ldap-example.org
```

Comment:
I think it is better to comment this out, as it is a very sensitive topic to send data to an LDAP server.